widgetastic_manageiq/expression_editor.py

```diff
@@ -102,10 +102,9 @@ class date_relative_form_view(View):  # noqa
     ROOT = "//div[@id='exp_editor_div']"
     MAKE_BUTTON = "//span[not(contains(@style,'none'))]//img[@alt='{}']"
     ATOM_ROOT = "./div[@id='exp_atom_editor_div']"
-    EXPRESSIONS_ROOT = VersionPick({
-        Version.lowest(): "./fieldset/div",
-        "5.10": ".//div[@class='panel-body']"
-    })
+    EXPRESSIONS_ROOT = VersionPick(
+        {Version.lowest(): "./fieldset/div", "5.10": ".//div[@class='panel-body']"}
```

Comment:
This is unfortunate, these are very nice to see on individual lines to be able to quickly visually parse which versions are supported by the pick.